ert3/engine/_record.py

```diff
@@ -1,10 +1,9 @@
-import ert3
-from ert3.engine import _utils
-
-from pathlib import Path
 import json
 from typing import Optional, TextIO
-import yaml
+from pathlib import Path
+
+import ert3
+from ert3.engine import _utils
```

Comment:
Maybe I contradicted myself when I said we should keep the relative importing to the `__init__.py` files? Maybe it does make sense to use it everywhere, considering the new information from our Microsoft Fellow.

Reply:
I'm thinking relative imports makes sense whenever things from the current (sub)module is imported. This is changed to a relative import in one of the later commits that I have now pushed...